packages/emotion/README.md

```diff
@@ -47,65 +33,324 @@ const myStyle = css`
+        transformOrigin: 'center bottom'
+      })
+    >
+  )
+}
+
+render(<Avatar src={logoUrl} />)
+```
 
-This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
-<a href="graphs/contributors"><img src="https://opencollective.com/emotion/contributors.svg?width=890&button=false" /></a>
+## cx
 
-## Backers
+`cx` is emotion's version of the popular [`classnames` library](https://github.com/JedWatson/classnames).
+`cx` is useful when combining multiple class names, even class names from your stylesheets and emotion generated class names.
 
-Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/emotion#backer)]
+**Key Features**
 
-<a href="https://opencollective.com/emotion#backers" target="_blank"><img src="https://opencollective.com/emotion/backers.svg?width=890"></a>
+- High performance integration with emotion
+- Custom class names. e.g, `.my-bem--class`, are appended in order.
+- Combines the actual content of emotion generated class names. Multiple emotion generated class names are input and a unique class name is output.
 
-## Sponsors
+**API**
 
-Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [[Become a sponsor](https://opencollective.com/emotion#sponsor)]
+**`cx`** - `(...args<string|number|function|object|array>): string`
 
-<a href="https://opencollective.com/emotion/sponsor/0/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/0/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/1/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/1/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/2/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/2/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/3/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/3/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/4/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/4/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/5/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/5/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/6/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/6/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/7/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/7/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/8/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/8/avatar.svg"></a>
-<a href="https://opencollective.com/emotion/sponsor/9/website" target="_blank"><img src="https://opencollective.com/emotion/sponsor/9/avatar.svg"></a>
+`cx` takes any number of arguments and returns a string class name.
```

Comment:
Why string class name? Is there another type of class name that we would be referring to here?